suites/nautilus/rgw/tier_0_rgw.yaml

```diff
@@ -1,3 +1,19 @@
+# add "test-config" object to add custom io config for tests
+# example: 
+#  - test:
+#     config:
+#       script-name: test_name.py
+#       config-file-name: config_fname.yaml
+#       timeout: 300
+#       test-config:
+#         user_count: 1
+#         bucket_count: 10
+#         objects_count: 5
+#         objects_size_range:
+#           min: 5M
+#           max: 15M
+# refer associated test configs in ceph-qe-sripts/rgw/ for config structure 
+
```

Comment:
IMO, we could add this information to `sanity_rgw.py` test module instead of each test suite... I am okay to have the document string here if we plan to use them in these suites

Reply:
ok. i will add in the `.py` files

Reply:
done